api_docs/changelog.md

```diff
@@ -1689,11 +1689,18 @@ No changes; feature level used for Zulip 5.0 release.
 
 **Feature level 76**
 
-* [`POST /fetch_api_key`](/api/fetch-api-key), [`POST
-  /dev_fetch_api_key`](/api/dev-fetch-api-key): The HTTP status for
-  authentication errors is now 401. This was previously 403.
-* All API endpoints now use the HTTP 401 error status for API requests
-  involving a deactivated user or realm. This was previously 403.
+* [`POST /fetch_api_key`](/api/fetch-api-key),
+  [`POST /dev_fetch_api_key`](/api/dev-fetch-api-key): The HTTP status
+  for authentication errors is now 401. These previously used the HTTP
+  403 error status.
+* [All REST API endpoints](/api/rest-error-handling): API requests that
+  involve a deactivated user or organization now use the HTTP 401 error
+  status. These previously used the HTTP 403 error status.
+* [All REST API endpoints](/api/rest-error-handling): All error responses
+  now include the `code` key with a machine-readable string value. The
+  default value for this key is `"BAD_REQUEST"`. As of this feature level,
+  clients should check the `code` key when looking for specific error
+  conditions.
```

Comment:
I think we always had the opinion that you should look at `code` to identify specific error conditions; what changed is that we started providing a standard `BAD_REQUEST` value for all API responses, rather than encoding "we haven't allocated a specific code to this failure" via absence of a `code`.

Reply:
Make sense. Updated as such.